crates/biome_cli/src/execute/traverse.rs

```diff
@@ -242,13 +249,33 @@ fn init_thread_pool() {
 
 /// Initiate the filesystem traversal tasks with the provided input paths and
 /// run it to completion, returning the duration of the process
-fn traverse_inputs(fs: &dyn FileSystem, inputs: Vec<OsString>, ctx: &TraversalOptions) -> Duration {
+fn traverse_inputs(
+    fs: &dyn FileSystem,
+    inputs: Vec<OsString>,
```

Comment:
Since this is not a reference, does this function clones/copies the vector? Since vec doesn't implement copy trait, how does it work perf. wise?

Reply:
No, AFAIK it doesn't copy the vector because the ownership of `inputs` is passed to the closure (it's passed by value), and `inputs` isn't used anymore after the closure.

Reply:
So, it basically transfers the ownership, not clones or copies it. Good to know!